packages/template-ui/src/components/ChannelHeader.vue

```diff
@@ -6,29 +6,38 @@
   >
     <template v-slot:default>
       <div class="align-items-start d-flex justify-content-between mt-3">
-        <h1>{{ section.title }}</h1>
-        <ChannelLogo :channel="channel" size="lg" />
       </div>
       <b-row>
-        <b-col md="6" sm="12">
+        <b-col xs="12" sm="8" md="9" lg="9" xl="8">
+          <h1 class="d-md-none h3">
+            {{ section.title }}
+          </h1>
+          <h1 class="d-md-block d-none">
+            {{ section.title }}
+          </h1>
           <div class="lead mb-2 text-muted">
             {{ headerDescription }}
           </div>
         </b-col>
+        <b-col v-if="displayLogoInHeader" class="d-none d-sm-flex justify-content-end">
```

Comment:
:+1: Other options is to have a `:class="{ d-none: displayLogoInHeader }"` instead of the v-if. But it looks better this way.